ddtrace/tracer/remote_config.go

```diff
@@ -75,6 +76,27 @@ func (t *tags) toMap() *map[string]interface{} {
 	return &m
 }
 
+func (t *tracer) dynamicInstrumentationRCUpdate(u remoteconfig.ProductUpdate) map[string]state.ApplyStatus {
+
+	applyStatus := map[string]state.ApplyStatus{}
+
+	for k, v := range u {
+		log.Debug("Received dynamic instrumentation RC configuration for %s\n", k)
+		applyStatus[k] = state.ApplyStatus{State: state.ApplyStateUnknown}
+		passFullConfiguration(k, string(v))
+	}
+
+	// XXX: This function would typically respond with a status to say if the config was acknowledged
+	// but since we're extracting from BPF, we do not have a feedback loop and
```

Comment:
need to finish the sentence?

Reply:
Thanks for catching, the comment was out of date anyway